tdrs-backend/tdpservice/data_files/static/admin/js/admin/admin_datafile_model.js

```diff
@@ -0,0 +1,33 @@
+$(window).on('load', function() {
+    //your code here
+    console.log('loaded');
+    var submitBtn=document.querySelector('button[type=submit]');    // add the first listener
+    var theForm = submitBtn.parentNode.parentNode;
+
+    for (var i = 0; i < theForm.childNodes.length; i++) {
+        console.log(theForm.childNodes[i].className)
+        if (theForm.childNodes[i].className == "actions") {
+          form_header = theForm.childNodes[i];
+          break;
+        }        
+    }
+    for (var i = 0; i < form_header.childNodes.length; i++) {
+        console.log(form_header.childNodes[i].className)
+        if (form_header.childNodes[i].className == "action-counter") {
+          number_of_files = form_header.childNodes[i];
+          break;
+        }        
+    }
+    submitBtn.addEventListener('click', function(e) {
+        e.preventDefault();
+        disableFields();
```

Comment:
getting an error when clicking the submit button

<img width="571" alt="image" src="https://github.com/user-attachments/assets/38c6c5f7-cccb-4b4e-97c9-3be941b357da">

Reply:
should be fixed now